src/pages/login/index.tsx

```diff
@@ -64,7 +68,8 @@ const handleEmailStage = async (
       props: {
         csrfToken,
         emailAddress,
-        emailError: "Enter a valid email address"
+        emailError: "Enter a valid email address",
+        serviceMessages: JSON.parse(JSON.stringify(serviceMessages))
```

Comment:
What's going on here? Why do we need to `JSON.parse(JSON.stringify(...))`? That results in a no-op?

Reply:
So the problem here is that we can't serialise a `Date()` object?

In that case, what is returned from `JSON.parse()` when you parse a date string? Can we use that as the type in our ServiceMessage instead?

Doing a `JSON.parse(JSON.stringify(...))` just feels like a bit of a workaround... 🤷

Reply:
Sorry for the late reply:

It seems that this is an issue with JS and it does not like serializing anything but scalar types for performance reasons: https://github.com/vercel/next.js/issues/13209#issuecomment-633149650. We have an existing stringify line somewhere else in the code. JSON.parse() returns an object with keys and stuff.